src/components/StakingVaults/EarnOpportunities.tsx

```diff
@@ -38,7 +38,9 @@ export const EarnOpportunities = ({ assetId, accountId }: EarnOpportunitiesProps
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   if (!asset) throw new Error(`Asset not found for AssetId ${assetId}`)
 
-  const stakingOpportunities = useAppSelector(selectAggregatedEarnUserStakingOpportunities)
+  const stakingOpportunities = useAppSelector(
+    selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
```

Comment:
Fixes the default validator not being present in case you're not staked to it already